rlberry/utils/tests/test_sb_agent.py

```diff
@@ -0,0 +1,54 @@
+import tempfile
+
+from stable_baselines3 import A2C
+
+from rlberry.envs import gym_make
+from rlberry.utils.sb_agent import StableBaselinesAgent
+from rlberry.utils.check_agent import (
+    check_agent_manager,
+    check_seeding_agent,
+    check_save_load,
+)
+
+
+def _sb3_check_rl_agent(agent_cls, env, init_kwargs=None):
+    """
+    Check that an RL agent can be instantiated and fit.
+
+    Note
+    ----
+    Does not check that the fit is additive, because that doesn't seem to be
+    always the case with StableBaselines3.
```

Comment:
this seems to be due to the callback that can stop the fit early (e.g. https://stable-baselines.readthedocs.io/en/master/guide/callbacks.html `_on_step` method). Would it make sense to redefine the default stablebaselines callback in order to avoid stopping early ? And is this early stopping a problem for Hyperparameter optimization ? (also @omardrwch on this)

Reply:
After looking into it, I think the default is no early stopping in stablebaselines (I had to go back several inherited modules in stablebaselines, this is quite the maze). The problem seems to be seomewhere else.

On the other hand, the learn method has  a loop that says 
`for update in range(1, total_timesteps // self.n_batch + 1):` which would not give a total of  `total_timesteps` iterations unless `self.n_batch` divides `total_timesteps`. Maybe the problem comes from here ?